### ISSUE TITLE
A11y plugin improvements

### ISSUE THREAD

**Issue by juliemr** [angular/protractor#1854]
- [ ] pending https://github.com/angular/protractor/issues/1842 - audit after each test and merge results
- [ ] Look into adding custom audits specific to Angular applications
- [x] Enable Tenon.io auditing for users of that service
- [x] Run the tests against Protractor's own website
- [x] Add an option to configure whether warnings from Chrome a11y are treated as pass or fail
- [ ] Lower priority: Add meaningful duration numbers to test output
- [x] More sophisticated truncating of long element strings

@marcysutton did I miss anything?

**Comment by marcysutton** [comment]
- [ ] Missing accessibility/successConfig in test.js

Question: will a11y plugin failures fail CI builds? Seems like part of #1842.

**Comment by juliemr** [comment]
Yup, plugin failures will fail CI builds - they change the exit code to non-zero, which is all that Travis cares about.

For merging results, I think it's up to what we feel is most useful. We should probably use the test name to help developers pinpoint where failures actually occurred, but do some sort of deduplication if the same issue is present in a bunch of tests.

**Comment by marcysutton** [comment]
I think I got the "identifying header" part solved by adding text to the Pass/Fail descriptions. It's especially useful when you run both integrations at the same time. PR coming shortly.

**Comment by marcysutton** [comment]
Hey @juliemr I have another improvement to add to the accessibility plugin: make the output color configurable. The red and green colors are very hard to see for some folks.

**Comment by sjelin** [comment]
There's a limited amount we can do with that.  Most of the output comes from tools like jasmine.

**Comment by dylanb** [comment]
@juliemr I would like to integrate the axe-core https://github.com/dequelabs/axe-core ruleset into this functionality - WDYT?

**Comment by juliemr** [comment]
Feature requests should  now go to https://github.com/angular/protractor-accessibility-plugin
